src/frontend/magnify/src/services/users/UserRepository.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { MagnifyApi } from '../http/http.service';
+import { UsersRepository } from './users.repository';
+
+describe('UserRepository', () => {
+  it('login success, test if token are well set in Axios', async () => {
+    const response = await UsersRepository.login('test', 'test');
+    expect(response.auth.access).not.toBe(null);
+    expect(response.auth.refresh).not.toBe(null);
+    expect(MagnifyApi.defaults.headers.common['Authorization']).toEqual(
+      `Bearer ${response.auth.access}`,
+    );
+  });
+  it('logout success, test if tokens are null', async () => {
+    UsersRepository.logout();
```

Comment:
In order to assert that access & refresh token are null after log out IMO you should test before logging out that they are _ not _ null

```
    expect(response.auth.access).not.toBe(null);
    expect(response.auth.refresh).not.toBe(null);
```

( Because we want to assert that caling `logout` correctly wipes tokens, so we need to tell the difference )